model.js

```diff
@@ -8,13 +9,54 @@ exports.selectTopics = () => {
 }
 
 exports.selectArticleById = (id) =>{
-    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`).then(({rows}) =>{
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`,[id]).then(({rows}) =>{
         if (rows.length === 0){
             return Promise.reject({
                 status: 404,
-                msg: "Couldn't find article"
+                msg: "Not found"
             })
         }
         return rows[0]
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = () =>{
+   const getArticles = () =>{ return db.query(`SELECT * FROM articles ORDER BY created_at DESC`).then(({rows}) =>{
+
+        
+        rows.forEach((article) =>{
+            article.comment_count = 0
```

Comment:
Comments _do_ already exist on many of the articles, but here you are responding with no comments, look into doing a JOIN statement to join the comment table and an aggregate function to count the comment on that article. This is a must before moving onto the frontend. If you need further clarification, you can send me a DM. 🙂